ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -291,6 +291,7 @@ static int HandleVnodeOperation(
         {
             if (FileFlagsBitIsSet(currentVnodeFileFlags, FileFlags_IsEmpty))
             {
+                KextLog_VnodeOp(currentVnode, vnodeType, procname, action, "Enumerating directory");
```

Comment:
I plan to drop these changes in the kext before merging in. This is just to demonstrate that it is working. With these log statements in place, I see the following outputs when enumerating and hydrating:

```
(3: 1534195840999) Note: Enumerating directory. Proc name: ls. Directory vnode action:  
    KAUTH_VNODE_READ_ATTRIBUTES 
    KAUTH_VNODE_READ_SECURITY 
     (vnode path: '/Users/sanoursa/GVFSTest/src')
(4: 1534195858026) Note: Hydrating file. Proc name: bash. File vnode action:  
    KAUTH_VNODE_READ_ATTRIBUTES 
    KAUTH_VNODE_READ_SECURITY 
     (vnode path: '/Users/sanoursa/GVFSTest/src/nuget.config')
```